ChartsDemo/Objective-C/Components/BalloonMarker.swift

```diff
@@ -42,11 +42,47 @@ open class BalloonMarker: MarkerImage
     
     open override func offsetForDrawing(atPoint point: CGPoint) -> CGPoint
     {
-        let size = self.size
-        var point = point
-        point.x -= size.width / 2.0
-        point.y -= size.height
-        return super.offsetForDrawing(atPoint: point)
+        var offset = self.offset
```

Comment:
So you just port the new marker for both Objc and Swift? Any new content based on #1741 ?
seems good to go

Reply:
Yea, I didn't make any changes to what #1741 provided. Just wanted to get the example merged and the author wasn't responding.